src/client/client.cpp

```diff
@@ -309,10 +309,10 @@ void Client::Stop()
 	if (m_localdb) {
 		infostream << "Local map saving ended." << std::endl;
 		m_localdb->endSave();
+		m_localdb.reset();
```

Comment:
Any reason to never delete `m_localdb`?  
Here looked like a good place to me to do so, but I am not fully sure